node_modules/@okta/i18n/package.json

```diff
@@ -1,40 +1,32 @@
 {
   "_args": [
     [
-      {
-        "raw": "@okta/i18n@18.7.0",
-        "scope": "@okta",
-        "escapedName": "@okta%2fi18n",
-        "name": "@okta/i18n",
-        "rawSpec": "18.7.0",
-        "spec": "18.7.0",
-        "type": "version"
-      },
-      "/Users/mauricio.castillosilva/okta/signin-widget"
+      "@okta/i18n@19.13.0",
+      "/Users/astadnikov/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@18.7.0",
-  "_id": "@okta/i18n@18.7.0",
+  "_from": "@okta/i18n@19.13.0",
+  "_id": "@okta/i18n@19.13.0",
   "_inCache": true,
+  "_installable": true,
```

Comment:
this has given trouble before, so if Travis has an error about not being able to resolve i18n, you might want to consider removing it